vagga.yaml

```diff
@@ -0,0 +1,25 @@
+containers:
+    gdg_dev:
+      setup:
+      - !Ubuntu trusty
+      - !UbuntuUniverse
+      - !Install [software-properties-common, npm, curl, git, openssl, libssl-dev, libpq-dev, ca-certificates, build-essential, wget, python-pip, python-dev, mercurial]
+      - !AptTrust keys: [5BB92C09DB82666C]
+      - !UbuntuPPA fkrull/deadsnakes
+      - !Install [python3.5]
+      - !Py3Requirements requirements/dev.txt
+      - !NpmInstall [nodejs, bower]
+      volumes:
```

Comment:
Indeed, however it was done for testing purposes. And someone told me not to push all the changes into one commit;)